### ISSUE TITLE
libmad from pacman and libogc conflict

### ISSUE THREAD

**Issue by Jan200101** [devkitPro/pacman-packages#216]
both the pacman repository ([here](https://github.com/devkitPro/pacman-packages/tree/master/ppc/libmad)) and libogc ([here](https://github.com/devkitPro/libogc/tree/master/libmad)) conflict with each other, providing the same lib name but not the same functions.
One example of such are all the MP3Player_* functions that are missing in the non-libogc version

**Comment by WinterMute** [comment]
The stock templates place the correct libogc paths ahead of the portlibs paths so they shouldn't conflict. Work is still ongoing to clean up libogc and eventually remove the libraries that shouldn't have been added to the libogc source tree in the first place but, until then, it's best to ensure libogc library and include files are in the search paths ahead of portlibs paths.

**Comment by Jan200101** [comment]
That is true, as of https://github.com/devkitPro/wii-examples/commit/56fd9dcc25c723e27e5f8213dd77889c0446b4ad libogc should be prioritised over portlibs
in my case I was working on older homebrew that used an older template

**Comment by hanssonson3** [comment]
Hi, sorry for hijacking this issue, but creating new issues doesn't seem to be working.

I've been wondering, I remember there being a repository hosting the PKGBUILD files for the various tools in the `dkp-<os>` repository, where did it go?
